### DISCUSSION TITLE
Update based on date - where clause

### DISCUSSION THREAD

**Discussion by var-che** [prisma/prisma#17381] [Q&A]
Hello

Is there a way to update an entry based on the where clause, that depends on checking the already existing date and the current one?
For example:

Post
storeId | updatedAt | title
51        | 1.13.2023   |  Original title

Updating with:
```
prisma.post.update({
  where: {updatedAt: { gt: 1.10.2023 }, storeId: 51},
  data { title: "updated title" }
}) // Intention code
```
In my experience, I had some trouble putting `updatedAt DateTime   @default(now())  @unique` 

Any tips on how to do this?

Edit: I have updated the post, specifying the way that where I can specifically target an entry with the unique key `storeId`

**Comment by nurul3101** [comment]
Hello @var-che 👋 
The records need to be uniquely identifiable in order for prisma client to update it.
In your case theoretically there could be many records that have updatedAt time value greater than 1.10.2023.
Do you want to update them all?

**Reply by var-che** [reply]
Thanks for the answer!
So, I actually have a way to uniquely target specific entries, by including keys that are unique to the entry. I did not present it in my original post. 
Basically, I already have a way to target an entry, AND I want to know if the `updatedAt` is greater than some value.

**Reply by nurul3101** [reply]
Hey @var-che,
Apologies for delay in getting back, this discussion slipped through the cracks.
In this case, you would need to use [updateMany](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#updatemany) instead of update.

**Comment by var-che** [comment]
Bump

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
